website/docs/cdktf/cli-reference/commands.mdx

```diff
@@ -676,3 +676,52 @@ Add a provider from a private registry to the project.
 ```bash
 $ cdktf provider add registry.example.com/acme/my-provider
 ```
+
+## provider add
```

Comment:
can we add this to the top of the page as well? :) we've got a linked list up there. Actually, we can probably delete that now because the devportal has a nice sidecar with all the headings in it
<img width="201" alt="Screen Shot 2022-10-31 at 12 24 06 PM" src="https://user-images.githubusercontent.com/83350965/199058347-e2fda4c1-db01-44a8-b736-1b131495eba0.png">

Reply:
@DanielMSchmidt can you also adjust this page so that the commands are in alphabetical order? I feel like this one and the one before it need to be placed differently on the page to make that happen. that will make it easier for users to find what they're looking for

Reply:
Sorting them would make sense, but I'd rather do that in a second PR (or would making sub-pages be nicer?)

Reply:
Should this be `# provider upgrade`?